frontend/packages/dev-console/integration-tests/features/e2e/helm-e2e.feature

```diff
@@ -0,0 +1,17 @@
+Feature: Helm Chart E2E 
+    As a user, I should be able to perform end to end scenarios related to Helm Chart
+
+
+Background:
+    Given user is at developer perspective
+    And user has selected namespace "aut-helm-e2e"
+
+
+@e2e
+Scenario: Helm E2E scenarios
+    Given user is at Add page
+    When user installs the Helm Chart
+    And user navigates to Helm Page
+    And user opens the Sidebar for Helm Chart
+    And user performs actions on the Helm Chart
+    Then user will not see Helm Chart
```

Comment:
```suggestion
    When user navigates to Topology page
    And user upgrades Helm Chart "nodejs-ex-k"
    And user uninstalls the Helm Chart "nodejs-ex-k"
    Then user will not see Helm Chart "nodejs-ex-k" in Topology page
    And user will not see Helm Chart in helm page
```

Reply:
Updated as per suggestions. Also added statements for Helm page and sidebar for Helm chart.